Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -367,35 +364,34 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
     }
 
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        if let currentSection = SearchListSection(rawValue: indexPath.section) {
-            switch currentSection {
-            case .searchSuggestions:
-                // heightForRowAtIndexPath is called *before* the cell is created, so to get the height,
-                // force a layout pass first.
-                suggestionCell.layoutIfNeeded()
-                return suggestionCell.frame.height
-            default:
-                return super.tableView(tableView, heightForRowAt: indexPath)
-            }
-        }
-
-        return 0
+        return super.tableView(tableView, heightForRowAt: indexPath)
     }
 
     override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         return 0
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = super.tableView(tableView, cellForRowAt: indexPath)
         switch SearchListSection(rawValue: indexPath.section)! {
         case .searchSuggestions:
-            suggestionCell.imageView?.image = searchEngines.defaultEngine.image
-            suggestionCell.imageView?.isAccessibilityElement = true
-            suggestionCell.imageView?.accessibilityLabel = String(format: .SearchSearchEngineSuggestionAccessibilityLabel, searchEngines.defaultEngine.shortName)
-            return suggestionCell
-
+            if let site = suggestions?[indexPath.row] {
+                if let cell = cell as? TwoLineTableViewCell {
+                    if Locale.current.languageCode == "en" {
```

Comment:
- Wondering if we can cleanup this if let chaining with something like below
`if let site = suggestions?[indexPath.row], let cell = cell as? TwoLineTableViewCell {`